app.js

```diff
@@ -10,11 +10,13 @@ const config = {
     PUBLIC_KEY: process.env.PUBLIC_KEY || undefined,
 };
 
+// Globals
+const LEETCODE_URL = "https://leetcode.com"
+
```

Comment:
nit: it would be better to have a file with all these globals instead of having them squeezed in this file